docs/compilers/Visual Basic/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -0,0 +1,8 @@
+# This document lists known breaking changes in Roslyn after .NET 6 all the way to .NET 7.
+
+## `RequiredMembersAttribute` cannot be manually applied
```

Comment:
All the errors that were introduced are breaking changes: types with required members no longer fulfill the `New()` constraint and cannot be inherited, etc

Reply:
There was already next to no chance you could inherit from a type with required members in VB: only constructors with `SetsRequiredMembersAttribute` could be called, as VB reports errors for using a constructor without it.